types/controller/agent_name.pp

```diff
@@ -0,0 +1,2 @@
+#Type for supported Agent Name of Controller Agent
```

Comment:
```suggestion
# @summary Type for supported Agent Name of Controller Agent
```
that will fix the REFERENCE.md. Please regenerate it after applying this change.